src/adhocracy_core/adhocracy_core/workflows/standard.py

```diff
@@ -10,9 +10,11 @@
     'states': {
         'draft': {'title': 'Draft',
                   'description': 'This phase is for internal review.',
-                  'acm': {'principals':         ['anonymous',   'participant', 'moderator', 'creator', 'initiator'],
+                  'acm': {'principals':         ['anonymous',   'moderator',   'participant', 'creator', 'initiator'],
                           'permissions':
-                          [['view',              'Deny',        'Deny',        'Allow',     'Allow',   'Allow'],
+                          [['view',              'Deny',        'Allow',       'Deny',        'Allow',   'Allow'],
+                           ['create_document',    None,         'Allow',        None,          None,      None],
+                           ['edit_document',      None,         'Allow',        None,         'Allow',   'Allow'],
```

Comment:
IMHO moderators should also be allowed to create/edit documents in later phases. Can you please clarify this?